### ISSUE TITLE
World stuck on generating 100%

### ISSUE THREAD

**Issue by VLeviathans** [Iskallia/Vault-public#324]

**Comment by Reed022** [comment]
Could you please let us know what launcher you’re using, and what version of VH? :) I suspect you’re using MultiMC, several people have had problems with it. Might be good to switch to CurseForge if so

**Comment by hom3gr0wn** [comment]
Im having the same issue and im using curse forge VH 1.2

**Comment by VLeviathans** [comment]
i was using multimc, curse doesnt work for me as an "error occors" when I try to load up the game

**Comment by Reed022** [comment]
Hmm okay, well oddly enough, I’ve heard that for some people it was as simple as updating the map mod. There’s two, Xaero’s Minimap and Xaero’s Fullscreen Map (or something like that)

**Comment by bmemike** [comment]
How much memory have you allocated? You want to give VH 6-8G.

**Comment by hom3gr0wn** [comment]
i've tried 6gb and 8gb same thing, i've updated the map mod same thing just sit at 100% and does nothing

**Comment by BONNe** [comment]
May be you could post a log file or crash file? It is hard to know where it is stuck without any content.

**Comment by hom3gr0wn** [comment]
[latest.log](https://github.com/Iskallia/Vault-public/files/6527450/latest.log)

**Comment by hom3gr0wn** [comment]
it's not really giving me any error logs just hangs at 100% every other mod pack I've tried works fine.

**Comment by hom3gr0wn** [comment]
Im not sure if this helps but it keeps saying this world is using experimental settings every time other mod packs dont do this so im thinking theres a mod or 2 out of date but not sure if thats the issue.

**Comment by BONNe** [comment]
Experimental setting warning is expected, as the vaults are in different dimension. 

Have you installed any extra mods? Or is it just a VaultHunters pack?
Have you tried to create a different seed? Or it is an issue with every world you try to create?

**Comment by hom3gr0wn** [comment]
i've tried removing vault hunters and made sure nothing was left behind and reinstalled it, same thing i've tried diffrerent worlds just doesnt load past that 100% screen.
only other mod pack I have is all the mods 6 but i've even tried removing that.

**Comment by BONNe** [comment]
In the modpack there are 107 mods...

**Comment by hom3gr0wn** [comment]
yeah everything is there I haven't touched it the first time i installed it it said minimap was out of date but updating it did nothing so I uninstalled everything minecraft and started over

**Comment by hom3gr0wn** [comment]
yeah says 107 file but when launching it says 108, when I check the folder it shows 106 resource pack says 107 so something not right just no idea how to fix it.

**Comment by BONNe** [comment]
No, it's all ok. 

Hmm, I have no idea how it could stuck at world generation without any crashes or errors. That is weird. 
You could try if flat word generates?

**Comment by hom3gr0wn** [comment]
same thing on superflat 100%

**Comment by hom3gr0wn** [comment]
[hs_err_pid6668.log](https://github.com/Iskallia/Vault-public/files/6528861/hs_err_pid6668.log)
not sure if this will help but when I close it this is what i get for error log

**Comment by hom3gr0wn** [comment]
not sure if this matters but the modding folder that's selected shows onedrive which is supposed to be impossible but when I try and select the curseforge folder manmually it says can't use onedrive...lol

**Comment by BONNe** [comment]
My last suggestion could be to update Java. I do not know if that is or could be an issue, but that is the only thing I could see as an issue here.

**Comment by hom3gr0wn** [comment]
I have uninstalled java 3 times and reinstalled it. I keep everything up to date I check regularly and why would this mod pack be the only 1 not working? somethings off in the modpack itself i can run any other modpack no problem

**Comment by hom3gr0wn** [comment]
I finally got it to work I had to wipe out my one drive uninstall curse/overwolf and start over stupid microsoft and their onedrive crap

**Comment by anumerical** [comment]
Having same issue. Using MS10 for OS. Using the forge app. It hangs at 100% generated. Then after staying there it closes. Error message; 
"The game crashed whilst rendering screen
Error: java.lang.OutOfMemoryError: GC overhead limit exceeded
Exit Code: -1"
CPU Utilization at near 100%. This is the modpack with all mods nothing else installed. Done through curseforge app 


[crash-2022-01-29_22.37.03-client.txt](https://github.com/Iskallia/Vault-public-S1/files/7966816/crash-2022-01-29_22.37.03-client.txt)
[debug.log](https://github.com/Iskallia/Vault-public-S1/files/7966817/debug.log)

**Comment by anumerical** [comment]
https://www.youtube.com/watch?v=fj-HabCAD0c
Followed this guide and was able to get it to work by updating ram allocation in profile in curseforge

**Comment by revot28** [comment]
> 

how much ram did you allocate? i did 10k and im still having the issue where im stuck at 100% i tried to update the maps earlier and it just wouldnt let me run the mod so i uninstalled and reinstalled everything...i dont even know if i downloaded the correct ones....

**Comment by revot28** [comment]
> https://www.youtube.com/watch?v=fj-HabCAD0c Followed this guide and was able to get it to work by updating ram allocation in profile in curseforge

i did this an allocated likie 10k and now it works!

**Comment by hom3gr0wn** [comment]
Turned out my issues was Microsoft one drive was saving the instance on the
cloud and it kept searching for it. Disable one drive and make sure it's
not saving to the cloud.

**Comment by Lurkerno341** [comment]
I'm having the same issue. Its only my survival world not loading, my creative world loads just fine. I checked and the files are saved to my hard drive, not my one drive and I've allocated 7072 mb to the game. By the way I'm using curse forge
[latest.log](https://github.com/Iskallia/Vault-public-S1/files/8308318/latest.log)
